src/analyzer/expr/expression_identifier.rs

```diff
@@ -213,6 +219,62 @@ pub fn get_functionlike_id_from_call(
     }
 }
 
+pub fn get_id_from_call(
```

Comment:
I'd prefer making this `get_functionlike_id_from_call` and then migrating the existing `get_functionlike_id_from_call` to `get_static_functionlike_id_from_call`

Reply:
Ok updated all these. I wasn't sure what callers to move over but let me know what you think